src/lib/storage.js

```diff
@@ -0,0 +1,32 @@
+/* global browser */
```

Comment:
nit: You shouldn't need `/* global browser */` here since we define the `browser: true` env in the root .eslintrc.json file.